lib/next-amp/README.md

```diff
@@ -215,14 +215,28 @@ Better than Option 1, but makes things more complicated.
 
 ### amp-bind
 
-For `amp-bind` expressions, the `amp-bind` extension is automatically imported using the same approach as for `amp-fx-collection`. The following will work out-of-the-box:
+AMP bindings can be expressed using the the `bindX` short-form instead of `data-amp-bind-x`. A nice
+side-effect is that most JS IDEs will offer component specific code-completion once you've
+typed `bind`.
+
+Here is an example binding an `AmpImg`'s `src` attribute via `bindSrc`:
+
+```
+<AmpImg width="300" height="200" src="image.png" bindSrc="imageUrl"/>
+<button on="tap:AMP.setState({ imageUrl: 'anotherImage.png' })">Change image</button>
+```
+
+Unfortunately, this doesn't work for non-AMP tags such as `div` or `input`. In this case you need to
+wrap the element inside the `AmpBind` component:
 
 ```
-<div data-amp-bind-text="hello"/>
-<button on="tap:AMP.setState({ "hello": "world" })">
+<AmpBind bindText="greeting">
```

Comment:
It is a pity extra markup is required find bind attributes, but if there is no alternative supported by Next.js then a wrapper element names sense.

Reply:
I'm not happy about it either. I see two other options:

1. HTML Element specific components:
    ```<AmpDiv bindText/>```
   This would have the added benefit of element specific auto completion. But it's probably more confusing than it helps. 
2. Add `bindX` types to the HTML elements and rename the attributes in AMP Optimizer. This would work best, but the downside is that it means we have to move Next.js specific features into AMP Optimizer.